Utils/Dataflow/pyDKB/storages/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""
+pyDKB.storages
+"""
+
+import importlib
+
+
+import exceptions
+import client
+
+
+_scope = None
+
+
+def setScope(scope):
+    """ Set default scope to look for storages.
+
+    :param scope: scope name
+    :type scope: str
+    """
+    global _scope
+    _scope = getScope(scope)
+
+
+def getScope(scope):
+    """ Initialize storages scope for further usage.
+
+    :param scope: scope name
+    :type scope: str
+    """
+    try:
+        full_name = __name__ + "." + scope
+        scope = importlib.import_module(full_name)
+    except ImportError:
+        raise exceptions.StorageException("Scope not defined: '%s'" % scope)
+    return scope
+
+
+def getClient(name, scope=None):
+    """ Get client for a given storage.
+
+    :raise StorageException: failed to get client by given name and scope
+
+    :param name: storage name
+    :type name: str
+    :param scope: scope name. If not specified, default value set with
+                  `setScope()` is used
+    :type scope: str, NoneType
+
+    :return: storage client
+    :rtype: client.Client
+    """
+    if scope:
+        scope = getScope(scope)
+    else:
+        scope = _scope
+    if scope is None:
+        raise exceptions.StorageException("Storages scope not specified")
```

Comment:
```suggestion
        raise exceptions.StorageException("Storage's scope not specified")
```

Reply:
It is not a mistake (or, better to say, it is a *different* mistake): the message says that scope of storages, from which should be picked one (or maybe "storage lookup scope"?), is not specified. "Storage scope" is more like a scope of a single storage -- which wouldn\`t have much sense in this situation.

Will be changed, thank you!